src/main/java/net/minestom/server/world/biomes/Biome.java

```diff
@@ -72,6 +72,7 @@ public static Builder builder() {
                 element.setFloat("scale", scale);
                 element.setFloat("downfall", downfall);
                 element.setString("category", category.name().toLowerCase(Locale.ROOT));
+                element.setByte("has_precipitation", (byte) 1); //todo bad fix
```

Comment:
Could address this quickly with another member variable in this class, unless you have other things in mind to do.